lookbook/docs/components/sub-header.md.erb

```diff
@@ -6,19 +6,33 @@ This component is an own implementation for OpenProject requirements of contextu
 
 ## Anatomy
 
-**Filter input:** (Optional) It will change depending on the viewport:
+### Desktop
 
-- **Desktop:** A basic [text input](https://primer.style/components/text-input) with a [search icon](https://primer.style/components/text-input#with-leading-and-trailing-visuals) which is mostly used on index pages to filter the elements below.
-- **Mobile:**
-  - There will be an IconButton shown instead of the input
-  - Clicking that, will show the input again and hide all other elements of the SubHeader
-  - An additional cancel button is shown to return to the previous state
+The sub header accepts a number of additional components. **All buttons** within the SubHeader require an icon to be shown, either as leading icon or as an icon-only variant.
 
-**Filter button:** (Optional)A button which is shown next to the filter input (or on the very left, if there is no filter input)
+On the left side:
 
-**Text:** (Optional) A bold text shown in the middle of the SubHeader
+- **Search filter**: (Optional) A basic [text input](https://primer.style/components/text-input) with a [search icon](https://primer.style/components/text-input#with-leading-and-trailing-visuals) which is mostly used on index pages to filter the elements below.
+- **Filter button:** (Optional) A button which is shown next to the filter input (or on the very left, if there is no filter input)
+- **Segmented controls** (optional) to switch two pre-filtered views
 
-**Action buttons**: (Optional) A set of buttons that are shown on the right side
+At the center of the sub header:
```

Comment:
Consistency error that happens in some other places as well: I think it would be better to use SubHeader in all places.